Apps/generator/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-tokio = { version = "1.21.2", features = ["full"] }
+tokio = { version = "1.21.2", features = ["rt-multi-thread", "time", "macros"] }
```

Comment:
Don't quite remember, do we still need tokio. On Wednesday you mentioned something about it having a massive footprint (guess that's why you changed from `full` to more specific features) and we don't actually need it?

Reply:
> Can't sadly comment directly in the files, but those are my further notes:
> 
> * config.rs > What does pps stand for (points/packages per second?). Maybe you can write it out.
**done**
> * config.rs (line 50-55) > write out `formatter` instead of `f` for better readability.
**done** 
> * all files > Sometimes `Err(_e)`, sometimes `Err(e)`?
**I use _ now if we don't care about the error value.**
> * all files > sometimes you have a ; at the end of the line, sometimes you don't (especially at the last line of functions. Is this something about rust?
**Yes it's rust. lines without semicolons are the return value.**
> 
> Apart from those, I'd say it looks very good :)

Thanks for the comments!

Reply:
> Don't quite remember, do we still need tokio. On Wednesday you mentioned something about it having a massive footprint (guess that's why you changed from `full` to more specific features) and we don't actually need it?
Yes, this was my attempt at reducing the executable size as Mr. Wieland mentioned the generator was super large. While playing around with it I realized the size is actually not too bad. Have to ask him about it again.

Reply:
Perfect, thanks for your response and clarifications :)